src/app-interoperability/application-package-definition.linkml.yaml

```diff
@@ -0,0 +1,497 @@
+        description: Statement about the changes for this application's release. The file should either be a markdown or PDF file.
+        rank: 40
+        range: string
+      site:
+        description: Link to the application's website.
+        rank: 50
+        range: string
+      tagline:
+        description: The application's slogan.
+        rank: 60
+        range: string
+      tags:
+        description: An array of strings that can be used to provide additional context for the application in a user interface to assist with task such as categorizing, searching, etc.
+        rank: 70
+        range: string
+        multivalued: true
+
+  Author:
+    description: Information about the application's author.
+    rank: 40
+    attributes:
+      name:
+        description: The name of the application's creator.
+        rank: 10
+        range: string
+      email:
+        description: Email address of the application's creator.
+        rank: 20
+        range: string
+        pattern: .*@[a-z0-9.-]*
+
+  Organization:
+    description: Information about the providing organization.
+    rank: 50
+    attributes:
+      name:
+        description: Organization responsible for the application's development and distribution.
+        rank: 10
+        range: string
+        required: true
+      site:
+        description: Link to the organization's website.
+        rank: 20
+        range: string
+
+  DeploymentProfile:
+    description: Represents a deployment configuration for the application.
+    rank: 60
+    slots:
+      - type
+      - components
+
+  HelmDeploymentProfile:
+    is_a: DeploymentProfile
+    rank: 40
+    slot_usage:
+      type:
+        equals_string: "helm.v3"
+        rank: 10
+      components:
+        range: HelmComponent
+        rank: 20
+
+  DockerComposeDeploymentProfile:
+    is_a: DeploymentProfile
+    rank: 40
+    slot_usage:
+      type:
+        equals_string: "docker-compose"
+        rank: 10
+      components:
+        range: DockerComposeComponent
+        rank: 20
+
+  Component:
+    description: A class representing a component of a deployment profile.
+    rank: 50
+    attributes:
+      name:
+        description: >-
+          A unique name used to identify the component package. For helm installations the name will be used as the chart name.
+          The name must be lower case letters and numbers and MAY contain dashes. 
+          Uppercase letters, underscores and periods MUST NOT be used.
+        required: true
+        range: string
+        rank: 10
+      properties:
+        description: >-
+           A dictionary element specifying the component packages's deployment details. 
+           See the [Component Properties](#component-properties) section below.
+        range: ComponentProperties #Arne: error in spec, as it is defined as map[string][interface{}] 
+        required: true
+        multivalued: true
+        inlined: true
+        inlined_as_list: false
```

Comment:
@arne-broering `ComponentProperties` (declared below) is an object with all the properties, so there is no need to have a list here. Although the answer to [this question](https://github.com/Silvanoc/margo-specification/pull/1/files#r1886787585) might change it...

```suggestion
```